pom.xml

```
@@ -54,22 +54,24 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Java Version Dependencies -->
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
 
         <!-- Arquillian Dependencies -->
-        <version.arquillian>1.7.0.Alpha14</version.arquillian>
-        <payara.arquillian.container.version>3.0.alpha8</payara.arquillian.container.version>
+        <version.arquillian>1.9.5.Final</version.arquillian>
+        <payara.arquillian.container.version>3.1</payara.arquillian.container.version>
```

For Payara 7 this should probably be 4.0.alpha2
```suggestion
        <payara.arquillian.container.version>4.0.alpha2</payara.arquillian.container.version>
```

---

The `payara-client-ee9` dependency will need updating to `payara-client-ee11`